public/app/features/explore/Wrapper.tsx

```diff
@@ -77,7 +109,7 @@ class WrapperUnconnected extends PureComponent<Props> {
     }
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps: Props) {
```

Comment:
`prevProps` is unused
```suggestion
  componentDidUpdate() {
```